app/src/test/java/ca/ualberta/CMPUT3012019T02/alexandria/ImageMessageTests.java

```diff
@@ -0,0 +1,74 @@
+package ca.ualberta.CMPUT3012019T02.alexandria;
+
+import org.junit.Test;
+
+import java.util.Date;
+import android.graphics.Bitmap;
+
+import ca.ualberta.CMPUT3012019T02.alexandria.model.Book;
+
```

Comment:
You can add import static org.junit.Assert.assertEquals;
Then you only need to call assertEquals() not Asset.assertEquals